commands/command_migrate.go

```diff
@@ -125,6 +125,21 @@ func rewriteOptions(args []string, opts *githistory.RewriteOptions, l *tasklog.L
 	}, nil
 }
 
+// isSpecialGitRef checks if a ref spec is a special git ref to exclude from
+// --everything
+func isSpecialGitRef(refspec string) bool {
+	parts := strings.SplitN(refspec, "/", 3)
+	if len(parts) < 3 {
+		return false
+	}
+	prefix := strings.Join(parts[:2], "/")
+	switch prefix {
+	case "refs/notes", "refs/bisect", "refs/replace":
```

Comment:
I think we'll also want to include `refs/stash` here, since we won't want to rewrite stashes.

Reply:
> 

Added a small unit test, I'll probably need a bit more time to figure out the how to get the test right in `.sh` unless people want to give some help on generating examples of excluded refs.

Reply:
I think this is probably fine as it stands.